m2m-transfers/V3/m2m-transfers-json-contract-v3.md

```diff
@@ -376,18 +376,42 @@ EDITTRAN - Edit existing transaction (expire existing transfer & create a new tr
 
 ### Error Codes
 
-| errorCode | loggingErrorMessage                       |
-| --------- | ----------------------------------------- |
-| 501       | Config file read / validation error       |
-| 502       | Invalid source account                    |
-| 503       | Invalid recipient account                 |
-| 504       | Insufficient information                  |
-| 505       | Invalid input                             |
-| 506       | Error creating/deleting recipient         |
-| 507       | Error creating/updating/deleting transfer |
-| 508       | Undefined error                           |
-| 509       | Member unverified                         |
-| 510       | AccountId incorrect                       |
+*See Logging Error Message section for additional details
```

Comment:
Should we be modeling this after the change address changes you made where there is a modified column.   Seems like the error code model is similar where you have error codes that have sub-errors.

Reply:
Added a Modified section for M2M JSon Error Codes section